src/mappings/Controller/schema.graphql

```diff
@@ -80,6 +81,13 @@ type ControllerRemoveGlobalConstraint @entity {
 	isPre: Boolean
 }
 
+type ContinuousLocking4ReputationParams @entity {
+	id: ID!
```

Comment:
total rep rewards ?

Reply:
`maxLockingBatches`. ?
`agreementHash`

Reply:
? there's no such field in the contract

Reply:
all these fields https://github.com/daostack/arc/blob/master/contracts/schemes/ContinuousLocking4Reputation.sol#L44-L55 
can be indexed .